app/assets/javascripts/discourse/lib/load-script.js.es6

```diff
@@ -86,22 +86,15 @@ export default function loadScript(url, opts) {
       cdnUrl = Discourse.CDN.replace(/\/$/, "") + url;
     }
 
-    // Some javascript depends on the path of where it is loaded (ace editor)
-    // to dynamically load more JS. In that case, add the `scriptTag: true`
-    // option.
-    if (opts.scriptTag) {
-      if (Ember.testing) {
-        throw new Error(
-          `In test mode scripts cannot be loaded async ${cdnUrl}`
```

Comment:
This (and `LOAD_ASYNC` in ace component) was introduced in https://github.com/discourse/discourse/commit/e66fa82f3a1d7fc769fe57b801d7f4c17d1b3e78. Since our test suite has been refactored to use `async/await`, I assumed this is no longer relevant? (plus tests still pass)

Reply:
I suspect this will still be a problem. `loadScript` happens in `didInsertElement` which doesn't not return a promise, so the test suite has no idea when the work has finished.

It's good the tests are passing, but I suspect it might be leaking promises/information. We could merge this optimistically and keep an eye on it.

Reply:
Should this become a problem in the future, do we fallback to using inline scripts in test environment? In that case, is there any other way to test `loadScript`'s behavior other than clicking around in a live instance?

Will merge this after the meetup.

Reply:
Yes I think if it becomes a problem in our automed test runners (jenkins/travis) we should revert back to the inline script.

I don't there is a good way to test it except manually. Let's give it a try and see what happens.